app/config.py

```diff
@@ -1 +1,7 @@
+import os
+
+
 DEBUG = True
+
+SQLALCHEMY_DATABASE_URI = os.getenv('DEED_DATABASE_URI',
```

Comment:
Nice catch. I also hadn't ran the database upgrade or passed this variable to the app. By-product of adding a DB to a frontend app.